src/annotate.rs

```diff
@@ -226,6 +246,48 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_kwctl_usage_is_added_when_annotations_is_none() -> Result<()> {
+        let dir = tempdir()?;
+
+        let file_path = dir.path().join("metadata.yml");
+        let mut file = File::create(file_path.clone())?;
+
+        let raw_metadata = format!(
```

Comment:
I don't think we have to convert that to a string, isn't a simple `&str` good enough for `write!`?